### DISCUSSION TITLE
Radsecproxy two LogDistination

### DISCUSSION THREAD

**Discussion by rasheedi99** [radsecproxy/radsecproxy#145] [Q&A] [answered]
Dears,

is it possible to add two log destinations in radsecproxy.conf as below ??
LogDestination        file:///var/log/radsecproxy/radsecproxy.log
LogDestination          x-syslog:///LOG_LOCAL0

since we need them to be in both files

**Comment by Janfred** [comment, marked as answer]
I would suggest to do this in the syslog configuration.
You can define multiple targets in your syslog-ng/rsyslog configuration for the given log source, which should enable you to achieve your goal.

Defining multiple `LogDestination`s in radsecproxy is not possible.

(Disclaimer: I'm not a radsecproxy maintainer)
